angr/storage/paged_memory.py

```diff
@@ -1058,4 +1076,26 @@ def unmap_region(self, addr, length):
             del self._pages[base_page_num + page]
             del self._symbolic_addrs[base_page_num + page]
 
+    def flush_pages(self, white_list):
+        """
+            :param white_list: white list of page number to exclude from the flush
+        """
+        white_list_page_number = []
+
+        for addr in white_list:
+            for page_addr in range(addr[0], addr[1], self._page_size):
+                white_list_page_number.append(page_addr / self._page_size)
```

Comment:
`page_addr // self._page_size`

I wonder if the code works at it is right now...